javascript/extractor/lib/typescript/package.json

```
@@ -2,7 +2,7 @@
     "name": "typescript-parser-wrapper",
     "private": true,
     "dependencies": {
-        "typescript": "5.2.2"
+        "typescript": "^5.3"
```

@erik-krogh Should this be an exact version like this?
```suggestion
        "typescript": "5.3.2"
```